src/frontends/onnx/tests/onnx_import_com_microsoft.in.cpp

```diff
@@ -405,6 +405,105 @@ NGRAPH_TEST(${BACKEND_NAME}, onnx_model_embed_layer_normalization_dynamic_shapes
+        0.36980811, 0.14644176, 0.56961840, 0.70373726, 0.28847644,
+    };
+    std::vector<float> segment_embeddings = {
+        0.09237389,
+        0.35404667,
+        0.55181628,
+        0.03362509,
+        0.96896178,
+        0.32099724,
+        0.22126268,
+        0.14126390,
+        0.09725992,
+        0.98404223,
+        0.26034093,
+        0.53702253,
+        0.44792616,
+        0.09956909,
+        0.35231167,
+    };
+    std::vector<float> gamma = {
+        0.46924916,
+        0.84114015,
+        0.90464777,
+        0.03755938,
+        0.50831544,
+    };
+    std::vector<float> beta = {
+        0.16684751,
+        0.77905101,
+        0.86493331,
+        0.41139671,
+        0.13997258,
+    };
+    std::vector<int> mask = {
+        1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 1, 1, 1, 0, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0,
+    };
+    std::vector<float> expected_output = {
+        -0.04089922, 0.35108989,  0.30442458,  0.39546335,  1.15422225,  0.10419128,  -0.19301927, 0.01070970,
+        0.43977541,  0.89119899,  -0.51436460, 1.99256825,  1.41077507,  0.38642293,  0.17583044,  0.03320138,
+        1.16508031,  -0.24356931, 0.47440714,  -0.17844005, 0.20463173,  1.90038323,  1.14138567,  0.34504607,
+        0.16403235,  -0.24976699, 0.29362509,  0.34502214,  0.41751838,  1.09390712,  0.12354189,  1.83025289,
+        1.05569196,  0.34413773,  0.35469764,  -0.69760042, 0.76338542,  1.75443077,  0.44126555,  0.18181801,
+        0.73277575,  0.45443264,  0.17068321,  0.36591727,  0.72869974,  -0.56090516, 0.14415455,  1.47314119,
+        0.42908576,  0.73084539,  -0.22373237, 2.26550221,  0.05606699,  0.39417523,  0.35234636,  0.78569502,
+        0.77521765,  -0.65131050, 0.40168875,  0.45527256,  0.38715565,  0.98521245,  2.21446753,  0.36345237,
+        -0.33269632, 0.36558092,  1.36846578,  1.37523413,  0.33698002,  0.28889543,  -0.40639281, 1.01643157,
+        0.59668219,  0.39197800,  1.03101778,  0.02551098,  -0.03612846, -0.01371557, 0.43444607,  0.96746695,
+        0.60583955,  -0.10362893, 0.40574494,  0.38046724,  0.87445319,  -0.00880148, -0.15437943, 0.08118075,
+        0.44650543,  0.85956848,  -0.27865338, 2.10837507,  0.04798460,  0.43948367,  -0.10185169, 0.19978794,
+        1.32323360,  1.20525467,  0.44288942,  -0.84200430, 0.52563053,  0.69949460,  0.73987913,  0.34668452,
+        0.74545687,  0.57696682,  0.22452033,  -0.27099937, 0.39649010,  0.87083614,  -0.18965788, 0.58206403,
+        -0.08108193, 0.42067638,  1.05117214,  -0.34287399, 0.20424896,  0.27994895,  0.46011117,  0.70890665,
+    };
+    std::vector<int> expected_mask_index = {
+        6,
+        5,
+        5,
+    };
+
+    auto test_case = test::TestCase(function, s_device);
+    test_case.add_input<int>(Shape{3, 8}, input_ids);
+    test_case.add_input<int>(Shape{3, 8}, segment_ids);
+    test_case.add_input<float>(Shape{10, 5}, word_embeddings);
+    test_case.add_input<float>(Shape{10, 5}, position_embeddings);
```

Comment:
It'd be good to have a test where position embeddings have different shape than word embeddings.

Reply:
thanks for your comment, i have updated the test case